demo/infra/app/variables.tf

```diff
@@ -30,5 +30,9 @@ variable "k8s_db_secret_count" {
   default = 50
 }
 
+variable "db_role" {
```

Comment:
Not 100% on this, but I think there is a `local.db_role` which is now unused due to this change that can be removed?

Reply:
Oops, need to drop `local.db_role`